src/frontends/lean/pp.cpp

```diff
@@ -127,13 +127,13 @@ void initialize_pp() {
     g_ellipsis_n_fmt  = new format(highlight(format("\u2026")));
     g_ellipsis_fmt    = new format(highlight(format("...")));
     g_placeholder_fmt = new format(highlight(format("_")));
-    g_lambda_n_fmt    = new format(highlight_keyword(format("\u03BB")));
+    g_lambda_n_fmt    = new format(highlight_keyword(format("\xCE\xBB"))); //\u03BB
     g_lambda_fmt      = new format(highlight_keyword(format("fun")));
```

Comment:
@nunoplopes Can we put the unicode character (λ) instead? g++ and clang++ have support for it.
```
    g_lambda_n_fmt    = new format(highlight_keyword(format("λ")));		
```
BTW, if VS does not support it, we will have to fix other places where we have used unicode characters directly.